src/ansys/fluent/core/workflow.py

```diff
@@ -477,44 +468,43 @@ def delete(self) -> None:
 
     def rename(self, new_name: str):
         """Rename the current task to a given name."""
-        if self._dynamic_interface:
-            self._command_source._dynamic_python_names = True
-            if (
-                self.python_name()
-                in self._command_source._repeated_task_python_name_display_text_map
-            ):
-                self._command_source._python_name_command_id_map[new_name] = (
-                    self._command_source._python_name_command_id_map.pop(
-                        self.python_name(), None
-                    )
-                )
-                self._command_source._python_name_display_id_map[new_name] = (
-                    self._command_source._python_name_display_id_map.pop(
-                        self.python_name(), None
-                    )
-                )
-                self._command_source._python_name_display_text_map.pop(
-                    self.python_name(), None
-                )
-                self._command_source._repeated_task_python_name_display_text_map.pop(
+        self._command_source._dynamic_python_names = True
+        if (
+            self.python_name()
```

Comment:
Shall we capture python_name in a local variable and use it throughout this function?

Reply:
Done, Thank you.